pkg/repo/mysql/services_integration_test.go

```diff
@@ -0,0 +1,213 @@
+package mysql
+
+import (
+	"context"
+	"log"
+	"testing"
+
+	"github.com/brianvoe/gofakeit/v6"
+	"github.com/google/uuid"
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/assert"
+	"go.expect.digital/translate/pkg/model"
+	"go.expect.digital/translate/pkg/repo"
+)
+
+var mysqlRepo *Repo
```

Comment:
alternative:
var repo *Repo

Reply:
Name conflict with package repo

Reply:
Alternative name `repository` to consider instead of `mysqlRepo`.